surveys/surveys.go

```diff
@@ -93,7 +93,9 @@ func GetAvailableSchemas() []LauncherSchema {
 		LauncherSchemaFromFilename("test_conditional_routing.json"),
 		LauncherSchemaFromFilename("test_confirmation_question.json"),
 		LauncherSchemaFromFilename("test_currency.json"),
-		LauncherSchemaFromFilename("test_date_range_period_validation.json"),
+		LauncherSchemaFromFilename("test_date_validation_combined.json"),
+		LauncherSchemaFromFilename("test_date_validation_range.json"),
+		LauncherSchemaFromFilename("test_date_validation_single.json"),
```

Comment:
@dwyeradam if we are gonna have more date validation tickets, do these need to be more descriptive? Or just deal with that hurdle when it comes

Reply:
I just wanted to have a uniform name without having the names so long as to be unwieldy. We can always rename if they get too generic.

Reply:
Want this to be merged now? or wait for SR PR to get to a stage where its ready, as charlotte might want to do PO approval on rest of date validation